<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winMain</class>
 <widget class="QMainWindow" name="winMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QffmpegUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabFile">
     <attribute name="title">
      <string>File</string>
     </attribute>
     <widget class="QPushButton" name="btnFileBrowser">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>7</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Files</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotalFiles">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>126</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 files selected.</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lstFileList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearList">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>32</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveUpFile">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>87</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Move Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteFile">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>108</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAdvance">
     <attribute name="title">
      <string>Advance</string>
     </attribute>
     <widget class="QPlainTextEdit" name="tbxAdvanceCmd">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>20</y>
        <width>370</width>
        <height>30</height>
       </rect>
      </property>
      <property name="plainText">
       <string>ffmpeg -version</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAdvanceCmd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>81</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Advance Command</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdvanceRun">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>3</y>
        <width>56</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txeAdvanceOutput">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>53</y>
        <width>370</width>
        <height>90</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbout">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QTextBrowser" name="tbrAbout">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff5500;&quot;&gt;QffmpegUI&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Version beta&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Update in April 2020&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;An opensource GUI tool developed under Python to deliver FFmpeg features.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;-----&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Repo: &lt;a href=&quot;https://github.com/wklchris/QffmpegUI&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;wklchris/QffmpegUI&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;License: LGPL v2.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
